react/features/overlay/components/OverlayContainer.js

```diff
@@ -0,0 +1,202 @@
+/* global APP */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PageReloadOverlay from './PageReloadOverlay';
+import SuspendedOverlay from './SuspendedOverlay';
+import UserMediaPermissionsOverlay from './UserMediaPermissionsOverlay';
+
+/**
+ * Implements a React Component that will display the correct overlay when
+ * needed.
+ */
+class OverlayContainer extends Component {
```

Comment:
At the moment his component only relies on props, there is no state nor lifecycle method usage so you can transform it into a stateless functional component.